src/control-plane/backend/lambda/api/service/quicksight/reporting-utils.ts

```diff
@@ -605,41 +606,51 @@ export function getFunnelTableVisualDef(visualId: string, viewName: string, even
       Width: '120px',
     });
 
-    const fieldIdRate = uuidv4();
     if (index === 0) {
-      groupBy.push({
-        NumericalDimensionField: {
-          FieldId: fieldIdRate,
-          Column: {
-            DataSetIdentifier: viewName,
-            ColumnName: 'rate',
-          },
+      continue;
+    }
+
+    const fieldIdRate = uuidv4();
+    groupBy.push({
```

Comment:
which test case(s) is validating those changes?

Reply:
this change is let 'rate' as the last column of the chart and rename to 'total_conversation_rate'.  we need check the generated json template if we wan't to check this change.